test/models/knowing_category_test.rb

```diff
@@ -6,4 +6,29 @@ class KnowingCategoryTest < ActiveSupport::TestCase
   # test "the truth" do
   #   assert true
   # end
+
+  test 'knowing category should be valid' do
+    knowing_category = FactoryBot.build(:knowing_category)
```

Comment:
```suggestion
    knowing_category = build(:knowing_category)
```
You can directly use the FactoryBot `build` method in tests because we've brought that capability into `ActiveSupport::TestCase`